server/client_manager.py

```diff
@@ -84,6 +84,11 @@ def send_host_message(self, msg):
         def send_motd(self):
             self.send_host_message('=== MOTD ===\r\n{}\r\n============='.format(self.server.config['motd']))
 
+        def send_player_count(self):
+            self.send_host_message('{}/{} players online.'.format(
+                self.server.get_player_count(),
+                self.server.config['playerlimit']))
```

Comment:
Wouldn't it be more practical to print an output like the `who` command in WoW/Minecraft/*nix? Otherwise this yields the same output as if I just opened up a new client and checked the player count from there.

Reply:
That was the intention. Doesn't /getarea already the same as `who`?